app/views/pages/assessments.html.erb

```diff
@@ -0,0 +1,71 @@
+<h1>Assessments</h1>
+
+<p>Assessments are ways to measure to what extent students are achieving the Student Learning Outcomes.</p>
+
+<p>The faculty were surveyed and asked to map the outcomes to their respective undergraduate classes. They were then asked which assessments would be used to measure those outcomes.  For your subject, you will see each assessment listed.  Each assessment will be used to measure how many students meet the outcomes selected by the faculty.</p>
```

Comment:
There's an extra space before `Each`.

It'd be easier to read this, and comment on specific things, if the lines wrapped at 80 characters